dor/providers/descriptor_generator.py

```diff
@@ -0,0 +1,37 @@
+from pathlib import Path
+from datetime import datetime, UTC
+
+from dor.settings import S, template_env
```

Comment:
I don't think we're using `S`, right? We'll have to think about the relationship between `settings` and `config` at some point, doesn't have to be now though.